### DISCUSSION TITLE
The difference between monospaced == true and Family::Monospace

### DISCUSSION THREAD

**Discussion by shi-yan** [pop-os/cosmic-text#120] [Q&A]
Hello there,

Thank you for this library. I'm trying to implement a terminal emulator using this library. For terminal emulator, I need to choose a monospaced font by default, but I also want to support unicode, and languages like Chinese etc.

I looked at the font fallback code, I saw this logic:

```rust
    let attrs = Attrs {
        color_opt: None,
        family: Family::Monospace,
        monospaced: true,
        stretch: Width::Normal,
        style: Style::Normal,
        weight: Weight::NORMAL,
        metadata: 0,
    };
    let fonts = font_system.get_font_matches(attrs);
```

I don't understand the difference between the two:
```rust
        family: Family::Monospace,
        monospaced: true,
```

if I set monospaced = true, I will find a nice monospaced font to use, however, other languages, such as Chinese, are not supported (unable to find a fallback font for them),

if I set monospaced = false, I will find a Chinese font, but the same Chinese font will also be selected for English characters.

Thanks,

**Comment by jackpot51** [comment]
I have removed `monospaced` from attrs now. Just use Family::Monospace
